.gitlab/kitchen_common/testing.yml

```diff
@@ -131,66 +131,67 @@
     KITCHEN_EC2_SUBNET: subnet-c18341ed
     KITCHEN_EC2_SG_IDS: sg-0f5617ceb3e5a6c39
 
+.kitchen_ec2_arm64:
+  variables:
+    KITCHEN_ARCH: arm64
+    KITCHEN_EC2_INSTANCE_TYPE: "t4g.xlarge"
+    # we need chef >= 15 for arm64 and for the arm? helper
+    CHEF_VERSION: 15.16.4
+  extends:
+    - .kitchen_ec2_location_us_east_1
+    - .kitchen_ec2_spot_instances
 
-# Kitchen: Test types (test suite * agent flavor + azure location)
+# Kitchen: Test types (test suite * agent flavor)
+# Don't specify location for the jobs as they can be ran either
+# in Azure or EC2, e.g. for ARM tests.
```

Comment:
Is there really a downside to specifying the `AZURE_LOCATION` environment variable in EC2 tests? I think it'll just get ignored with no effect. That'd avoid having to set the location in each individual test.